webserver/Controllers/CompanyController.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace webserver.Controllers;
 
-[Authorize(Roles=Common.BZE_Role+","+Common.Company_Role)]
```

Comment:
If you don't need them, drop them.
But I think they are necessary. We don't want unauthenticated users to access this controller, do we?

Reply:
Yeah i probably commented it out when testing/debugging. We do need it, so it's back now